lib/live_debugger_web/components.ex

```diff
@@ -72,6 +72,57 @@ defmodule LiveDebuggerWeb.Components do
     """
   end
 
+  @doc """
+  Renders an input with label.
+  """
+  attr(:id, :any, default: nil)
+  attr(:name, :any)
+  attr(:label, :string, default: nil)
+  attr(:value, :any)
+
+  attr(:type, :string,
+    default: "text",
+    values: ~w(checkbox color date datetime-local email file month number password
+               range search select tel text textarea time url week)
+  )
+
+  attr(:field, Phoenix.HTML.FormField, required: true)
+
+  attr(:rest, :global,
+    include: ~w(accept autocomplete capture cols disabled form list max maxlength min minlength
+                multiple pattern placeholder readonly required rows size step)
+  )
+
+  slot(:inner_block)
+
+  def input(%{field: %Phoenix.HTML.FormField{} = field} = assigns) do
+    assigns =
+      assigns
+      |> assign(field: nil, id: assigns.id || field.id)
+      |> assign_new(:name, fn -> field.name end)
+      |> assign_new(:value, fn -> field.value end)
```

Comment:
You usually want this field to accept only `Phoenix.HTML.FormField` as attribute that stores all arguments passed to input

Check input definition from here
https://hexdocs.pm/phoenix_live_view/form-bindings.html#form-events